java/ql/test/library-tests/frameworks/apache-commons-lang3/flow.ql

```diff
@@ -17,11 +29,18 @@ class Conf extends TaintTracking::Configuration {
 class HasFlowTest extends InlineExpectationsTest {
   HasFlowTest() { this = "HasFlowTest" }
 
-  override string getARelevantTag() { result = "hasTaintFlow" }
+  override string getARelevantTag() { result = ["hasTaintFlow", "hasValueFlow"] }
 
   override predicate hasActualResult(Location location, string element, string tag, string value) {
     tag = "hasTaintFlow" and
-    exists(DataFlow::Node src, DataFlow::Node sink, Conf conf | conf.hasFlow(src, sink) |
+    exists(DataFlow::Node src, DataFlow::Node sink, TaintFlowConf conf | conf.hasFlow(src, sink) |
+      sink.getLocation() = location and
+      element = sink.toString() and
+      value = "y"
```

Comment:
You could also match the empty string `""` here, see  #5347

Reply:
Though maybe it would actually be more useful to have the argument index as `value`. Currently the tests might be error-prone because it might not be easy to notice when the wrong argument is matched.